wallet-test/src/test/scala/org/bitcoins/wallet/WalletTest.scala

```diff
@@ -0,0 +1,28 @@
+package org.bitcoins.wallet
+
+import org.bitcoins.wallet.models.{AccountDAO, AddressDAO, MnemonicCodeDAO}
+import org.bitcoins.wallet.util.BitcoinSWalletTest
+
+class WalletTest extends BitcoinSWalletTest {
+
+  val accountDAO = AccountDAO(dbConfig, chainParams)
+  val addressDAO = AddressDAO(dbConfig, chainParams)
+  val mnemonicDAO = MnemonicCodeDAO(dbConfig)
+
+  it should "create a new wallet" in {
+    val passphrase = "foobar"
+
+    for {
+      _ <- Wallet.initialize(chainParams, passphrase)
```

Comment:
This test crashes. There's some weirdness going in different databases being connected to in setup (`beforeAll` of BitcoinSWalletTest`) and the actual tests being run